samples/tooling/profiles/profiles.cpp

```diff
@@ -30,14 +30,9 @@
 // The Vulkan Profiles library is part of the SDK and has been copied to the sample's folder for convenience
 #include "vulkan_profiles.hpp"
```

Comment:
Why is this header copied from the SDK? Wouldn't there be any version issues when the SDK is updated?

Reply:
No, it's the other way round. The one from the SDK might (and has) introduce breaking changes, so if we were to rely on that one, this sample might not work for everyone. Packaging it will avoid such breakages. Also our sample are suppose to work without any SDK installed.

Reply:
> Also our sample are suppose to work without any SDK installed.

I see... we're using what we have in third_party/vulkan/include. And there's nothing like a vulkan_profiles.hpp.
How do we get the vulkan stuff here, any why is that file missing?

Reply:
vulkan_profiles.hpp is only packaged with the SDK, it's not part of the Vulkan headers.

Reply:
Ah, ok. Thanks for that clarifications.